src/modules/helper.js

```diff
@@ -0,0 +1,186 @@
+import countHomeItems from './counter.js';
+import countComments from './countComments.js';
+
+const gameID = process.env.GAMEID;
+const commentForm = document.getElementById('comment-submit');
+
+// get comments of an astronaut
+const createCommentsSection = (usercomments) => {
+  const commentDiv = document.getElementById('comments-data');
+  usercomments.forEach((comment) => {
+    commentDiv.insertAdjacentHTML('beforeend', `<b class="comm">${comment.creation_date} : ${comment.username} : ${comment.comment} </b><br>`);
+    countComments();
+  });
+};
+
+export const getAllAstronautComments = async (astronautId) => {
+  try {
+    const req = await fetch(`https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/${gameID}/comments?item_id=${astronautId}`);
+    const data = await req.json();
+    if (!req.ok) {
+      return data;
+    }
+    const commentData = data;
+    createCommentsSection(commentData);
+    return commentData;
+  } catch (error) {
+    return error;
+  }
+};
+
+export const addNewComment = async (newComment) => {
+  try {
+    const req = await fetch(`https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/${gameID}/comments`, {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json',
+      },
+      body: JSON.stringify(newComment),
+    });
+
+    if (!req.ok) {
+      return req.status;
+    }
+    getAllAstronautComments(newComment.item_id);
+    return req;
+  } catch (error) {
+    return error;
+  }
+};
+
+commentForm.addEventListener('submit', (e) => {
+  e.preventDefault();
+  const name = document.getElementById('name');
+  const comment = document.getElementById('comment');
+  const itemComment = document.getElementById('item-id');
+
+  const newComment = {
+    item_id: itemComment.value,
+    username: name.value,
+    comment: comment.value,
+  };
+  name.value = '';
+  comment.value = '';
+  itemComment.value = '';
+
+  addNewComment(newComment);
+});
```

Comment:
- [ ] After submitting my comment, the comment was not updated on the screen. I would also suggest that you fix that for a good user experience.

![image](https://user-images.githubusercontent.com/98453979/205461367-5540d404-2591-4da4-8257-2aab7f729f94.png)

Reply:
The Comment section also updates after setting up the .env.

Procedure create a .env file and paste the "GAMEID = lv3bo7lVqNrhL88bp1Z3"

![image](https://user-images.githubusercontent.com/64089437/205461934-1039f9cd-ed4d-4c69-8a43-83979ad377ca.png)

Reply:
> The Comment section also updates after setting up the .env.
> 
> Procedure create a .env file and paste the "GAMEID = lv3bo7lVqNrhL88bp1Z3"
> 
> ![image](https://user-images.githubusercontent.com/64089437/205461934-1039f9cd-ed4d-4c69-8a43-83979ad377ca.png)

- [ ] Okey..that's great. Always remember to include such kind of instructions on your readme file